Cannot install env on Mac M1 due to numcodecs incompatibility

I cannot install the conda environment on my Mac M1 due to an incompatibility with the numcodecs package when it is installed from PyPI. 

output:

<details>

```
[...]
  Building wheel for elephant (setup.py): started
  Building wheel for elephant (setup.py): finished with status 'done'
  Created wheel for elephant: filename=elephant-0.12.0-py3-none-any.whl size=524768 sha256=80c25f283a09c9dbcee90a9ed8259f06d444eba9de90144dfdaabd50a2f8ae93
  Stored in directory: /Users/rly/Library/Caches/pip/wheels/ba/fa/88/93ed9aeee89bd686409e2a11aa74a9618994d4fc9d5246265b
  Building wheel for numcodecs (pyproject.toml): started
  Building wheel for numcodecs (pyproject.toml): finished with status 'error'
Successfully built python-docx biopython gevent lxml pyedflib pyintan futures neuroconv elephant
Failed to build numcodecs

Pip subprocess error:
  Running command git clone --filter=blob:none --quiet https://github.com/catalystneuro/neuroconv.git /private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-install-hn9ksb7q/neuroconv_0a8b25ea10e94a489a614d70d8b24495
  error: subprocess-exited-with-error
  
  × Building wheel for numcodecs (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [246 lines of output]
      /private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/config/pyprojecttoml.py:108: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
        warnings.warn(msg, _BetaConfiguration)
      [numcodecs] setting up Blosc extension
      [numcodecs] compiling Blosc extension with SSE2 support
      [numcodecs] compiling Blosc extension without AVX2 support
      [numcodecs] setting up Zstandard extension
      [numcodecs] setting up LZ4 extension
      [numcodecs] setting up compat extension
      [numcodecs] setting up shuffle extension
      [numcodecs] setting up vlen extension
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.macosx-11.0-arm64-cpython-39
      creating build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/lzma.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/astype.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/gzip.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/base64.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/bz2.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/version.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/compat.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/checksum32.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/ndarray_like.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/quantize.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/registry.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/pickles.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/fixedscaleoffset.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/__init__.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/bitround.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/zlib.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/shuffle.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/delta.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/zfpy.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/msgpacks.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/packbits.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/categorize.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/json.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      copying ./numcodecs/abc.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs
      creating build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_categorize.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_registry.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_vlen_utf8.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_bitround.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_gzip.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_delta.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_lzma.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_bz2.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/__init__.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_pickles.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_quantize.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_lz4.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_msgpacks.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_astype.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_packbits.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/common.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_base64.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_shuffle.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_fixedscaleoffset.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_compat.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_zlib.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_json.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_ndarray_like.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_vlen_array.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_zstd.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_vlen_bytes.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_entrypoints.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_checksum32.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_zfpy.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      copying ./numcodecs/tests/test_blosc.py -> build/lib.macosx-11.0-arm64-cpython-39/numcodecs/tests
      running egg_info
      writing numcodecs.egg-info/PKG-INFO
      /private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/dist.py:159: UserWarning: newlines not allowed and will break in the future
        warnings.warn("newlines not allowed and will break in the future")
      writing dependency_links to numcodecs.egg-info/dependency_links.txt
      writing requirements to numcodecs.egg-info/requires.txt
      writing top-level names to numcodecs.egg-info/top_level.txt
      listing git files failed - pretending there aren't any
      reading manifest file 'numcodecs.egg-info/SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE.txt'
      writing manifest file 'numcodecs.egg-info/SOURCES.txt'
      running build_ext
      building 'numcodecs.blosc' extension
      creating build/temp.macosx-11.0-arm64-cpython-39
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/blosc
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/lz4-1.9.3
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zlib-1.2.8
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zstd-1.4.8
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zstd-1.4.8/common
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zstd-1.4.8/compress
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zstd-1.4.8/decompress
      creating build/temp.macosx-11.0-arm64-cpython-39/c-blosc/internal-complibs/zstd-1.4.8/dictBuilder
      creating build/temp.macosx-11.0-arm64-cpython-39/numcodecs
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/rly/mambaforge/envs/env-electron-python/include -arch arm64 -fPIC -O2 -isystem /Users/rly/mambaforge/envs/env-electron-python/include -arch arm64 -DHAVE_LZ4=1 -DHAVE_ZLIB=1 -DHAVE_ZSTD=1 -Ic-blosc/blosc -Ic-blosc/internal-complibs/lz4-1.9.3 -Ic-blosc/internal-complibs/zstd-1.4.8 -Ic-blosc/internal-complibs/zlib-1.2.8 -Ic-blosc/internal-complibs/zstd-1.4.8/compress -Ic-blosc/internal-complibs/zstd-1.4.8/dictBuilder -Ic-blosc/internal-complibs/zstd-1.4.8/decompress -Ic-blosc/internal-complibs/zstd-1.4.8/legacy -Ic-blosc/internal-complibs/zstd-1.4.8/common -Ic-blosc/internal-complibs/zstd-1.4.8/dll -Ic-blosc/internal-complibs/zstd-1.4.8/deprecated -Ic-blosc/internal-complibs/zstd-1.4.8/dll/example -I/Users/rly/mambaforge/envs/env-electron-python/include/python3.9 -c c-blosc/blosc/bitshuffle-generic.c -o build/temp.macosx-11.0-arm64-cpython-39/c-blosc/blosc/bitshuffle-generic.o -msse2 -stdlib=libc++ -DSHUFFLE_SSE2_ENABLED
      clang: warning: argument unused during compilation: '-msse2' [-Wunused-command-line-argument]
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/rly/mambaforge/envs/env-electron-python/include -arch arm64 -fPIC -O2 -isystem /Users/rly/mambaforge/envs/env-electron-python/include -arch arm64 -DHAVE_LZ4=1 -DHAVE_ZLIB=1 -DHAVE_ZSTD=1 -Ic-blosc/blosc -Ic-blosc/internal-complibs/lz4-1.9.3 -Ic-blosc/internal-complibs/zstd-1.4.8 -Ic-blosc/internal-complibs/zlib-1.2.8 -Ic-blosc/internal-complibs/zstd-1.4.8/compress -Ic-blosc/internal-complibs/zstd-1.4.8/dictBuilder -Ic-blosc/internal-complibs/zstd-1.4.8/decompress -Ic-blosc/internal-complibs/zstd-1.4.8/legacy -Ic-blosc/internal-complibs/zstd-1.4.8/common -Ic-blosc/internal-complibs/zstd-1.4.8/dll -Ic-blosc/internal-complibs/zstd-1.4.8/deprecated -Ic-blosc/internal-complibs/zstd-1.4.8/dll/example -I/Users/rly/mambaforge/envs/env-electron-python/include/python3.9 -c c-blosc/blosc/bitshuffle-sse2.c -o build/temp.macosx-11.0-arm64-cpython-39/c-blosc/blosc/bitshuffle-sse2.o -msse2 -stdlib=libc++ -DSHUFFLE_SSE2_ENABLED
      clang: warning: argument unused during compilation: '-msse2' [-Wunused-command-line-argument]
      c-blosc/blosc/bitshuffle-sse2.c:20:4: error: SSE2 is not supported by the target architecture/platform and/or this compiler.
        #error SSE2 is not supported by the target architecture/platform and/or this compiler.
         ^
      In file included from c-blosc/blosc/bitshuffle-sse2.c:23:
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/emmintrin.h:14:2: error: "This header is only meant to be used on x86 and x64 architecture"
      #error "This header is only meant to be used on x86 and x64 architecture"
       ^
      In file included from c-blosc/blosc/bitshuffle-sse2.c:23:
      In file included from /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/emmintrin.h:17:
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/xmmintrin.h:14:2: error: "This header is only meant to be used on x86 and x64 architecture"
      #error "This header is only meant to be used on x86 and x64 architecture"
       ^
      In file included from c-blosc/blosc/bitshuffle-sse2.c:23:
      In file included from /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/emmintrin.h:17:
      In file included from /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/xmmintrin.h:17:
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:14:2: error: "This header is only meant to be used on x86 and x64 architecture"
      #error "This header is only meant to be used on x86 and x64 architecture"
       ^
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:54:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_vec_init_v2si(__i, 0);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:133:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_packsswb((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:163:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_packssdw((__v2si)__m1, (__v2si)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:193:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_packuswb((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:220:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpckhbw((__v8qi)__m1, (__v8qi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:243:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpckhwd((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:264:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpckhdq((__v2si)__m1, (__v2si)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:291:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpcklbw((__v8qi)__m1, (__v8qi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:314:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpcklwd((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:335:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_punpckldq((__v2si)__m1, (__v2si)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:356:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_paddb((__v8qi)__m1, (__v8qi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:377:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_paddw((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:398:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_paddd((__v2si)__m1, (__v2si)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:420:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_paddsb((__v8qi)__m1, (__v8qi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      /Library/Developer/CommandLineTools/usr/lib/clang/14.0.0/include/mmintrin.h:443:12: error: invalid conversion between vector type '__m64' (vector of 1 'long long' value) and integer type 'int' of different size
          return (__m64)__builtin_ia32_paddsw((__v4hi)__m1, (__v4hi)__m2);
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      fatal error: too many errors emitted, stopping now [-ferror-limit=]
      20 errors generated.
      [numcodecs] command '/usr/bin/clang' failed with exit code 1
      Traceback (most recent call last):
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/unixccompiler.py", line 185, in _compile
          self.spawn(compiler_so + cc_args + [src, '-o', obj] + extra_postargs)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/ccompiler.py", line 1041, in spawn
          spawn(cmd, dry_run=self.dry_run, **kwargs)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/spawn.py", line 70, in spawn
          raise DistutilsExecError(
      distutils.errors.DistutilsExecError: command '/usr/bin/clang' failed with exit code 1
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "<string>", line 258, in build_extension
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/command/build_ext.py", line 246, in build_extension
          _build_ext.build_extension(self, ext)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build_ext.py", line 548, in build_extension
          objects = self.compiler.compile(
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/ccompiler.py", line 600, in compile
          self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/unixccompiler.py", line 187, in _compile
          raise CompileError(msg)
      distutils.errors.CompileError: command '/usr/bin/clang' failed with exit code 1
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "/Users/rly/mambaforge/envs/env-electron-python/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/Users/rly/mambaforge/envs/env-electron-python/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/Users/rly/mambaforge/envs/env-electron-python/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 251, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 413, in build_wheel
          return self._build_with_temp_dir(['bdist_wheel'], '.whl',
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 398, in _build_with_temp_dir
          self.run_setup()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 284, in <module>
        File "<string>", line 275, in run_setup
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 108, in setup
          return distutils.core.setup(**attrs)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1221, in run_command
          super().run_command(command)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/normal/lib/python3.9/site-packages/wheel/bdist_wheel.py", line 343, in run
          self.run_command("build")
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1221, in run_command
          super().run_command(command)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build.py", line 131, in run
          self.run_command(cmd_name)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 1221, in run_command
          super().run_command(command)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "<string>", line 251, in run
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/command/build_ext.py", line 84, in run
          _build_ext.run(self)
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build_ext.py", line 345, in run
          self.build_extensions()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build_ext.py", line 467, in build_extensions
          self._build_extensions_serial()
        File "/private/var/folders/cp/nh4x2lls1qq03jh58crvh8t00000gp/T/pip-build-env-vuy97v3d/overlay/lib/python3.9/site-packages/setuptools/_distutils/command/build_ext.py", line 493, in _build_extensions_serial
          self.build_extension(ext)
        File "<string>", line 261, in build_extension
      __main__.BuildFailed
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for numcodecs
ERROR: Could not build wheels for numcodecs, which is required to install pyproject.toml-based projects

failed

CondaEnvException: Pip failed
```

</details>

However, first installing numcodecs from conda into an environment and then updating the environment using the `environment-MAC.yml` works.

MWE 1:
```
conda env create -f ./tools/anaconda-env/environment-MAC.yml
```

MWE 2:
```
mamba create --name env-electron-python --yes
mamba activate env-electron-python
# mamba install numcodecs  # this fixes it
mamba env update -n env-electron-python --file environment-MAC.yml
```

Suggested fix:
Pin `numcodecs = 0.11.0` in the conda section of `environment-MAC.yml`

--- a/tools/anaconda-env/environment-MAC.yml
+++ b/tools/anaconda-env/environment-MAC.yml
@@ -7,6 +7,7 @@ dependencies:
   - PyInstaller = 4.7
   - nodejs = 16.14.2
   - git = 2.34.1
+  - numcodecs = 0.11.0
   - pip
   - pip:
       - chardet == 4.0.0

